scripts/dashboard/WalletBalance.vue

```diff
@@ -344,6 +353,21 @@ function restoreWallet() {
                             "
                             >{{ primaryImmatureBalanceStr }}</span
                         >
+                        <div
+                            v-if="showImmatureBalanceIcon"
+                            class="immatureTooltip"
```

Comment:
Add class `ptr` on the Icon so that desktop users know it's clickable.

```suggestion
                            class="immatureTooltip ptr"
```

Then **LGTM**, looks good!